scripts/eosio_build_darwin.sh

```diff
@@ -1,406 +1,311 @@
-						sudo chown -R "$(whoami)" /usr/local/share
-					fi
-					"${XCODESELECT}" --install 2>/dev/null;
-					printf "\\tUpdating Home Brew.\\n"
-					if ! brew update
-					then
-						printf "\\tUnable to update Home Brew at this time.\\n"
-						printf "\\tExiting now.\\n\\n"
-						exit 1;
-					fi
-					printf "\\tInstalling Dependencies.\\n"
-					if ! "${BREW}" install --force ${DEP}
-					then
-						printf "\\tHomebrew exited with the above errors.\\n"
-						printf "\\tExiting now.\\n\\n"
-						exit 1;
-					fi
-                                        if [[ "$DEP" == "llvm@4" ]]; then
-                                                "${BREW}" unlink ${DEP}
-					elif ! "${BREW}" unlink ${DEP} && "${BREW}" link --force ${DEP}
-					then
-						printf "\\tHomebrew exited with the above errors.\\n"
-						printf "\\tExiting now.\\n\\n"
+if [ ! -d /usr/local/Frameworks ]; then
+	printf "\\n${bldred}/usr/local/Frameworks is necessary to brew install python@3. Run the following commands as sudo and try again:${txtrst}\\n"
+	printf "sudo mkdir /usr/local/Frameworks && sudo chown $(whoami):admin /usr/local/Frameworks\\n\\n"
+	exit 1;
+fi
+
+if [ $COUNT -gt 1 ]; then
+	printf "\\nThe following dependencies are required to install EOSIO:\\n"
+	printf "${DISPLAY}\\n\\n"
+	if [ $1 == 0 ]; then read -p "Do you wish to install these packages? (y/n) " answer; fi
+	case ${answer} in
+		1 | [Yy]* )
+			"${XCODESELECT}" --install 2>/dev/null;
+			if [ $1 == 0 ]; then read -p "Do you wish to update homebrew packages first? (y/n) " answer; fi
+			case ${answer} in
+				1 | [Yy]* )
+					if ! brew update; then
+						printf " - Brew update failed.\\n"
 						exit 1;
+					else
+						printf " - Brew update complete.\\n"
 					fi
-				break;;
-				[Nn]* ) echo "User aborting installation of required dependencies, Exiting now."; exit;;
-				* ) echo "Please type 1 for yes or 2 for no.";;
+				;;
+				[Nn]* ) echo "Proceeding without update!";;
+				* ) echo "Please type 'y' for yes or 'n' for no."; exit;;
 			esac
-		done
-	else
-		printf "\\n\\tNo required Home Brew dependencies to install.\\n"
-	fi
+			brew tap eosio/eosio # Required to install mongo-cxx-driver with static library
+			printf "\\nInstalling Dependencies...\\n"
+			# Ignore cmake so we don't install a newer version.
+			# Build from source to use local cmake; see homebrew-eosio repo for examples
+			# DON'T INSTALL llvm@4 WITH --force!
+			OIFS="$IFS"
+			IFS=$','
+			for DEP in $DEPS; do
+				# Eval to support string/arguments with $DEP
+				if ! eval $BREW install $DEP; then
+					printf " - Homebrew exited with the above errors!\\n"
+					exit 1;
+				fi
+			done
+			IFS="$OIFS"
+		;;
+		[Nn]* ) echo "User aborting installation of required dependencies, Exiting now."; exit;;
+		* ) echo "Please type 'y' for yes or 'n' for no."; exit;;
+	esac
+else
+	printf "\\n - No required Home Brew dependencies to install.\\n"
+fi
 
+printf "Checking CMAKE installation...\\n"
```

Comment:
Why is cmake being checked for twice in this script?

Reply:
Whoops, must have copied and pasted improperly